### ISSUE TITLE
Config values aren't actually used in code

### ISSUE THREAD

**Issue by Hidoni** [teamfusion/rottencreatures#25]
While writing the PR for #24 I realized this mod doesn't actually *use* any of the config values it loads, on both Fabric and Forge.  
The code defines and accesses config values through the `ConfigEntries` class defined in your code, but this class's fields never have their values written to by the side-specific mod loading code:
- On Forge, you have to access the values through the `ForgeConfigSpec.ConfigValue` instances that are returned when you use `ForgeConfigSpec.Builder.define()`, as those are the ones that Forge actually loads values into from the file.
- On Fabric, as you're using AutoConfig, you'll want to [get an instance of your Config class](https://shedaniel.gitbook.io/cloth-config/auto-config/reading-the-config) and read the values from that.

**Comment by ItsBlackGear** [comment]
Yes! i realized this as well too and i've been working on a proper config since the current one is not working, this should be fixed by next update ^^"
